packages/vinxi/lib/file-system-router.js

```diff
@@ -76,21 +77,18 @@ export class BaseFileSystemRouter extends EventTarget {
 	}
 
 	glob() {
-		if (os.platform() === "win32") {
-			return fg.convertPathToPattern(this.config.dir + "//**//*");
-		} else {
-			return (
-				join(this.config.dir, "**/*") + `.{${this.config.extensions.join(",")}}`
-			);
-		}
+		return posix.join(
+			fg.convertPathToPattern(normalize(this.config.dir)), 
+			"**/*"
+		) + `.{${this.config.extensions.join(",")}}`;
```

Comment:
The only important thing here is that the path should be normalized to posix style path before being passed to `convertPathTopattern` to prevent the following on Windows:

:x: without posix normalization
`\\(User)\\some\\path` => `/(User\\)/some/path`

✔️ with posix normalization
`\\(User)\\some\\path` => `/\\(User\\)/some/path`

Reply:
Apperently `{js}` does not work. To work with a single extension a trailing comma is required `{js,}`. Maybe there is a better way to express this.

Reply:
The goal of this PR is to get vinxi to run on Windows. Not fixing edge cases of fast-glob here. I will open a separate issue for this.